app-kotlin/src/main/kotlin/pwittchen/github/com/rxbiometric/MainActivity.kt

```diff
@@ -48,23 +49,17 @@ class MainActivity : AppCompatActivity() {
 
 
     button.setOnClickListener { _ ->
-      disposable = RxPreconditions
-        .canHandleBiometric(this)
-        .flatMapCompletable {
-          if (!it) Completable.error(BiometricNotSupported())
-          else
-            RxBiometric
-              .title("title")
-              .description("description")
-              .negativeButtonText("cancel")
-              .negativeButtonListener(DialogInterface.OnClickListener { _, _ ->
-                showMessage("cancel")
-              })
-              .cancellationSignal(CancellationSignal())
-              .executor(mainExecutor)
-              .build()
-              .authenticate(this)
-        }
+      disposable = RxBiometric
```

Comment:
Why `canHandleBiometric(...)` method is removed? Not all devices supports biometric.